modules/rosa-imds-machine-pools.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: CONCEPT
+[id="rosa-imds-machine-pools_{context}"]
+= Instance Metadata Service on machine pools
+
+Instance metadata allows you to configure and manage a running instance. Instance metadate also allows you to access user data that you need to launch your instance.
+
+IMDSv2 uses session-oriented requests. With session-oriented requests, you create a session token that defines the session duration, which can be a minimum of one second and a maximum of six hours. During the specified duration, you can use the same session token for subsequent requests. After the specified duration expires, you must create a new session token to use for future requests.
+
+There are two types of ways to access instance metadata from a running instance:
+
+* Instance Metadata Service Version 1 (IMDSv1) - a request/response method
+* Instance Metadata Service Version 2 (IMDSv2) - a session-oriented method
+
+When creating your ROSA cluster, you select to use either both IMDSv1 and IMDSv2 or specify that your cluster should only use IMDSv2. The instance metadata service distinguishes between IMDSv1 and IMDSv2 requests based on whether, for any given request, either the PUT or GET headers, which are unique to IMDSv2, are present in that request. If you specify to use IMDSv2 only, IMDSv1 ceases to function for your cluster. All machine pools on your cluster will use whichever IMDS type you select.
```

Comment:
I would farther clarify what the service is, how to reach it and where the mentioned "requests" and being sent to. It's purely AWS thing.

Reply:
Could you explain these pieces?

Reply:
I think the later added link to the AWS documentation is enough. The point was in giving a guiding how to use the metadata and that the service is not part of the cluster.